### ISSUE TITLE
Contradictory readme/examples + how to use tap in sync mode

### ISSUE THREAD

**Issue by caroline-d-lu** [singer-io/tap-klaviyo#13]
I am encountering a few issues:

### 1. Contradiction between README and repository's current config.sample.json file

The readme states that the config.json file should follow this format:

```
{
    "credentials": {
        "api_key": "pk_XYZ"
    },
    "start_date": "2017-01-01T00:00:00Z",
    "user_agent": "email_address",
}
```

Yet the example included in the repo has the following format:
```
{
    "api_key": "asasdkldasdlkaasdkfdsjadf",
    "start_date": "2017-01-01T00:00:00Z",
    "user_agent": "email_address"
}
```
_**Which config file format is correct?**_

### 2. Another contradiction between README and repository's current config.sample.json file

The readme also states that "The Klaviyo events you want to replicate each have a unique metric ID that needs to be added to the config file. Use -l option to list all available Klaviyo metrics in your account." This sentence seems to refer to the old format of the config file. 

Commit ad4fe087fdcc840edff1beedd499a1c180a9afb4 updated the config file, and the old format (below) included unique metric IDs. 

_**When using the repo in its current state, should the config file include metric IDs?**_

```
{
    "api_key": "asasdkldasdlkaasdkfdsjadf",
    "start_date": "2017-01-01T00:00:00Z",
    "user_agent": "email_address",
    "events": [
      {"name": "receive", "id": "ID", "primary_key": "uuid"},
      {"name": "click", "id": "ID", "primary_key": "uuid"},
      {"name": "open", "id": "ID", "primary_key": "uuid"},
      {"name": "bounce", "id": "ID", "primary_key": "uuid"},
      {"name": "unsubscribe", "id": "ID", "primary_key": "uuid"},
      {"name": "mark_as_spam", "id": "ID", "primary_key": "uuid"}
    ],
    "lists": [
      {"name": "lists", "primary_key": "id"}
    ],
    "exclusions": [
      {"name": "global_exclusions", "primary_key": "email"}
    ]
    "user_agent": "email_address"
}
```

### 3. Difficulty using tap in sync mode / unexpected output to STDOUT

I am attempting to execute the following command:

`tap-klaviyo --config config.json`

With the following config file:
```
{
    "api_key": "[api key here]",
    "start_date": "2012-01-01T00:00:00Z",
    "user_agent": "carolinelu4@gmail.com"
}
```
The result to stdout:
`INFO METRIC: {"metric": "http_request_duration", "type": "timer", "value": 0.17759299278259277, "tags": {"status": "succeeded", "endpoint": "metric_list", "http_status_code": 200}}`

According to the [sync mode documentation](https://github.com/singer-io/getting-started/blob/master/docs/SYNC_MODE.md), "During sync mode, the Tap is expected to emit Schema, Record, and State messages to stdout." **Why am I not seeing the schema, record, and state messages in stdout?**

In an attempt to remedy this issue, I have gone through my catalog.json file and I have made sure that all of the schema have "selected" set to "true." I have then run the following command:

`tap-klaviyo --config config.json --catalog catalog.json`

Again, I get the following result to stdout:
`INFO METRIC: {"metric": "http_request_duration", "type": "timer", "value": 0.18678712844848633, "tags": {"status": "succeeded", "endpoint": "metric_list", "http_status_code": 200}}`

_**Can anyone tell me what I am doing wrong in my attempts to operate the tap in sync mode?**_

Any assistance is much appreciated.

For reference, the contents of my catalog.json file are below.
```
{
  "streams": [
    {
      "key_properties": [
        "id"
      ], 
      "schema": {
        "selected": true,
        "type": "object", 
        "properties": {
          "event_properties": {
            "type": "object", 
            "properties": {}
          }, 
          "uuid": {
            "type": "string"
          }, 
          "event_name": {
            "type": "string"
          }, 
          "timestamp": {
            "type": "integer"
          }, 
          "object": {
            "type": "string"
          }, 
          "datetime": {
            "type": "string"
          }, 
          "person": {
            "type": "object", 
            "properties": {
              "email": {
                "type": "string"
              }
            }
          }, 
          "statistic_id": {
            "type": "string"
          }, 
          "id": {
            "type": "string"
          }
        }
      }, 
      "metadata": [
        {
          "breadcrumb": [], 
          "metadata": {
            "table-key-properties": "id", 
            "forced-replication-method": "INCREMENTAL"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "event_properties"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "uuid"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "event_name"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "timestamp"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "object"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "datetime"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "person"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "statistic_id"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "id"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }
      ], 
      "stream": "bounce", 
      "tap_stream_id": "M6rVU9"
    }, 
    {
      "key_properties": [
        "id"
      ], 
      "schema": {
        "selected": true,
        "type": "object", 
        "properties": {
          "event_properties": {
            "type": "object", 
            "properties": {}
          }, 
          "uuid": {
            "type": "string"
          }, 
          "event_name": {
            "type": "string"
          }, 
          "timestamp": {
            "type": "integer"
          }, 
          "object": {
            "type": "string"
          }, 
          "datetime": {
            "type": "string"
          }, 
          "person": {
            "type": "object", 
            "properties": {
              "email": {
                "type": "string"
              }
            }
          }, 
          "statistic_id": {
            "type": "string"
          }, 
          "id": {
            "type": "string"
          }
        }
      }, 
      "metadata": [
        {
          "breadcrumb": [], 
          "metadata": {
            "table-key-properties": "id", 
            "forced-replication-method": "INCREMENTAL"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "event_properties"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "uuid"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "event_name"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "timestamp"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "object"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "datetime"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "person"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "statistic_id"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "id"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }
      ], 
      "stream": "click", 
      "tap_stream_id": "PAw8nD"
    }, 
    {
      "key_properties": [
        "id"
      ], 
      "schema": {
        "selected": true,
        "type": "object", 
        "properties": {
          "event_properties": {
            "type": "object", 
            "properties": {}
          }, 
          "uuid": {
            "type": "string"
          }, 
          "event_name": {
            "type": "string"
          }, 
          "timestamp": {
            "type": "integer"
          }, 
          "object": {
            "type": "string"
          }, 
          "datetime": {
            "type": "string"
          }, 
          "person": {
            "type": "object", 
            "properties": {
              "email": {
                "type": "string"
              }
            }
          }, 
          "statistic_id": {
            "type": "string"
          }, 
          "id": {
            "type": "string"
          }
        }
      }, 
      "metadata": [
        {
          "breadcrumb": [], 
          "metadata": {
            "table-key-properties": "id", 
            "forced-replication-method": "INCREMENTAL"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "event_properties"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "uuid"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "event_name"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "timestamp"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "object"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "datetime"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "person"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "statistic_id"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "id"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }
      ], 
      "stream": "dropped_email", 
      "tap_stream_id": "NSRPqC"
    }, 
    {
      "key_properties": [
        "id"
      ], 
      "schema": {
        "selected": true,
        "type": "object", 
        "properties": {
          "event_properties": {
            "type": "object", 
            "properties": {}
          }, 
          "uuid": {
            "type": "string"
          }, 
          "event_name": {
            "type": "string"
          }, 
          "timestamp": {
            "type": "integer"
          }, 
          "object": {
            "type": "string"
          }, 
          "datetime": {
            "type": "string"
          }, 
          "person": {
            "type": "object", 
            "properties": {
              "email": {
                "type": "string"
              }
            }
          }, 
          "statistic_id": {
            "type": "string"
          }, 
          "id": {
            "type": "string"
          }
        }
      }, 
      "metadata": [
        {
          "breadcrumb": [], 
          "metadata": {
            "table-key-properties": "id", 
            "forced-replication-method": "INCREMENTAL"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "event_properties"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "uuid"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "event_name"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "timestamp"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "object"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "datetime"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "person"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "statistic_id"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "id"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }
      ], 
      "stream": "mark_as_spam", 
      "tap_stream_id": "MHsYwL"
    }, 
    {
      "key_properties": [
        "id"
      ], 
      "schema": {
        "selected": true,
        "type": "object", 
        "properties": {
          "event_properties": {
            "type": "object", 
            "properties": {}
          }, 
          "uuid": {
            "type": "string"
          }, 
          "event_name": {
            "type": "string"
          }, 
          "timestamp": {
            "type": "integer"
          }, 
          "object": {
            "type": "string"
          }, 
          "datetime": {
            "type": "string"
          }, 
          "person": {
            "type": "object", 
            "properties": {
              "email": {
                "type": "string"
              }
            }
          }, 
          "statistic_id": {
            "type": "string"
          }, 
          "id": {
            "type": "string"
          }
        }
      }, 
      "metadata": [
        {
          "breadcrumb": [], 
          "metadata": {
            "table-key-properties": "id", 
            "forced-replication-method": "INCREMENTAL"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "event_properties"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "uuid"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "event_name"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "timestamp"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "object"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "datetime"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "person"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "statistic_id"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "id"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }
      ], 
      "stream": "open", 
      "tap_stream_id": "MN6eSa"
    }, 
    {
      "key_properties": [
        "id"
      ], 
      "schema": {
        "selected": true,
        "type": "object", 
        "properties": {
          "event_properties": {
            "type": "object", 
            "properties": {}
          }, 
          "uuid": {
            "type": "string"
          }, 
          "event_name": {
            "type": "string"
          }, 
          "timestamp": {
            "type": "integer"
          }, 
          "object": {
            "type": "string"
          }, 
          "datetime": {
            "type": "string"
          }, 
          "person": {
            "type": "object", 
            "properties": {
              "email": {
                "type": "string"
              }
            }
          }, 
          "statistic_id": {
            "type": "string"
          }, 
          "id": {
            "type": "string"
          }
        }
      }, 
      "metadata": [
        {
          "breadcrumb": [], 
          "metadata": {
            "table-key-properties": "id", 
            "forced-replication-method": "INCREMENTAL"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "event_properties"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "uuid"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "event_name"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "timestamp"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "object"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "datetime"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "person"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "statistic_id"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "id"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }
      ], 
      "stream": "receive", 
      "tap_stream_id": "JdetRc"
    }, 
    {
      "key_properties": [
        "id"
      ], 
      "schema": {
        "selected": true,
        "type": "object", 
        "properties": {
          "event_properties": {
            "type": "object", 
            "properties": {
              "List": {
                "type": "string"
              }, 
              "$event_id": {
                "type": "string"
              }
            }
          }, 
          "uuid": {
            "type": "string"
          }, 
          "event_name": {
            "type": "string"
          }, 
          "timestamp": {
            "type": "integer"
          }, 
          "object": {
            "type": "string"
          }, 
          "datetime": {
            "type": "string"
          }, 
          "person": {
            "type": "object", 
            "properties": {
              "email": {
                "type": "string"
              }
            }
          }, 
          "statistic_id": {
            "type": "string"
          }, 
          "id": {
            "type": "string"
          }
        }
      }, 
      "metadata": [
        {
          "breadcrumb": [], 
          "metadata": {
            "table-key-properties": "id", 
            "forced-replication-method": "INCREMENTAL"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "event_properties"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "uuid"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "event_name"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "timestamp"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "object"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "datetime"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "person"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "statistic_id"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "id"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }
      ], 
      "stream": "subscribe_list", 
      "tap_stream_id": "KEaAXj"
    }, 
    {
      "key_properties": [
        "id"
      ], 
      "schema": {
        "selected": true,
        "type": "object", 
        "properties": {
          "event_properties": {
            "type": "object", 
            "properties": {}
          }, 
          "uuid": {
            "type": "string"
          }, 
          "event_name": {
            "type": "string"
          }, 
          "timestamp": {
            "type": "integer"
          }, 
          "object": {
            "type": "string"
          }, 
          "datetime": {
            "type": "string"
          }, 
          "person": {
            "type": "object", 
            "properties": {
              "email": {
                "type": "string"
              }
            }
          }, 
          "statistic_id": {
            "type": "string"
          }, 
          "id": {
            "type": "string"
          }
        }
      }, 
      "metadata": [
        {
          "breadcrumb": [], 
          "metadata": {
            "table-key-properties": "id", 
            "forced-replication-method": "INCREMENTAL"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "event_properties"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "uuid"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "event_name"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "timestamp"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "object"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "datetime"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "person"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "statistic_id"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "id"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }
      ], 
      "stream": "unsubscribe", 
      "tap_stream_id": "PvSS3f"
    }, 
    {
      "key_properties": [
        "email"
      ], 
      "schema": {
        "selected": true,
        "type": "object", 
        "properties": {
          "timestamp": {
            "type": "string"
          }, 
          "reason": {
            "type": "string"
          }, 
          "object": {
            "type": "string"
          }, 
          "email": {
            "type": "string"
          }
        }
      }, 
      "metadata": [
        {
          "breadcrumb": [], 
          "metadata": {
            "table-key-properties": "email", 
            "forced-replication-method": "FULL_TABLE"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "timestamp"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "reason"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "object"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "email"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }
      ], 
      "stream": "global_exclusions", 
      "tap_stream_id": "global_exclusions"
    }, 
    {
      "key_properties": [
        "id"
      ], 
      "schema": {
        "selected": true,
        "type": "object", 
        "properties": {
          "updated": {
            "type": "string"
          }, 
          "person_count": {
            "type": "integer"
          }, 
          "name": {
            "type": "string"
          }, 
          "created": {
            "type": "string"
          }, 
          "object": {
            "type": "string"
          }, 
          "type": {
            "type": "string"
          }, 
          "id": {
            "type": "string"
          }
        }
      }, 
      "metadata": [
        {
          "breadcrumb": [], 
          "metadata": {
            "table-key-properties": "id", 
            "forced-replication-method": "FULL_TABLE"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "updated"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "person_count"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "name"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "created"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "object"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "type"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }, 
        {
          "breadcrumb": [
            "properties", 
            "id"
          ], 
          "metadata": {
            "inclusion": "automatic"
          }
        }
      ], 
      "stream": "lists", 
      "tap_stream_id": "lists"
    }
  ]
}
```

**Comment by luandy64** [comment]
I think this was closed by #14, right @caroline-d-lu?

**Comment by caroline-d-lu** [comment]
Yes
